src/dscanner/analysis/unused.d

```diff
@@ -407,6 +412,8 @@ private:
 		{
 			if (!uu.isRef && tree.length > 1)
 			{
+			    if (uu.uncertain)
+			        continue;
 				immutable string certainty = uu.uncertain ? " might not be used."
 					: " is never used.";
```

Comment:
This line would need adapting as well if this is considered for inclusion.

Reply:
Sorry but i don't see the problem, can you explain more ?

Reply:
If we go with this change this line should be rewritten to `immutable string certainty = " is never used."; as "might" now no longer can happen.
Though to be honest it's quite a long time ago since I made this change.